File: VB/DevExpress.Calendar/IPage.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraBars

Namespace DevExpress.Calendar
	Public Interface IPage
		Sub Page_Load(ByVal e As PageLoadEventArgs)
		Sub Page_Invalidate()
	End Interface
End Namespace

File: CS/DevExpress.Calendar/IPage.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraBars;

namespace DevExpress.Calendar {
    public interface IPage {
        void Page_Load(PageLoadEventArgs e);
        void Page_Invalidate();
    }
}
